src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockImporter.scala

```diff
@@ -293,9 +293,9 @@ class BlockImporter(
     }
   }
 
-  private def importWith(importFuture: => Future[NewBehavior])(state: ImporterState): Unit = {
+  private def importWith(importTask: => Task[NewBehavior])(state: ImporterState): Unit = {
     context become running(state.importingBlocks())
-    importFuture.onComplete {
+    importTask.runToFuture.onComplete {
       case Failure(ex) => throw ex
```

Comment:
Who is going to handle this exception? The `onComplete` runs asynchronously. At least we should log it.